nextroute/factory/model.go

```diff
@@ -15,35 +15,19 @@ import (
 // must be built from the ground up.
 func NewModel(
 	input schema.Input,
-	modelOptions ModelOptions,
+	modelOptions nextroute.ModelOptions,
```

Comment:
As stated, these options need to be in the factory package, they influence the semantics of this NewModel factory function in this package.